### DISCUSSION TITLE
Can we estimate estimate the variance of VIMP this way？

### DISCUSSION THREAD

**Discussion by A-Pai** [kogalur/randomForestSRC#161] [Q&A] [answered]
Can we estimate estimate the variance of VIMP this way？
Permutation importance permutes a variable’s OOB data and compares the resulting OOB prediction error to the original OOB prediction error，the permutation is done randomly，so we can do this many times and can get a error difference value  each time，we can use this many error difference value to estimate the variance of VIMP，is it feasible?  Is  there anything wrong?

**Comment by ishwaran** [comment, marked as answer]
No that won't work.  In your proposal, the error difference values obtained are not statistically independent.  Therefore, it is not as simple as you say to get an estimate for the variance.  The way around this is to use U-statistics theory, which is what we did in the paper.
